src/args.ts

```diff
@@ -64,6 +63,12 @@ function getParams() {
         describe: 'The client secret, this allows you to encrypt the secret in your build configuration instead of storing it in a config file',
         type: 'string'
       },
+      env: {
+        alias: 'e',
+        describe: 'Override the mappings in config with environment variables.',
+        boolean: true,
+        default: false
+      },
```

Comment:
Added the `env` flag for the import script for parity, however, this time defaulting to false so the behavior remains the same.